src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -2276,7 +3058,7 @@ namespace WaterToAirHeatPumpSimple {
         while (true) {
```

Comment:
I never did like this iteration method just to get RatedLatentCapacity on iteration #1 just because SHReff needs a reference latent to calculate SHR for constant fan mode. This coil has a rated or reference latent capacity, just like it has a rated or reference total and sensible capacity. Would be nice to get rid of this at some point.